src/layers.jl

```diff
@@ -61,11 +61,15 @@ end
  Define action of layer on inputs: do a graph convolution, add this (weighted by convolutional weight) to the features themselves (weighted by self weight) and the per-feature bias (concatenated to match number of nodes in graph).
 
 # Arguments
-- input: AtomGraph object
+- input: a FeaturizedAtoms object, or graph_laplacian, encoded_features
+
+# Note
+In the case of providing two matrices, the following conditions must hold:
+- lapl must be square and of dimension N x N where N is the number of nodes in the graph
+- X must be of dimension M x N, where M is `size(l.convweight)[2]` (or equivalently, `size(l.selfweight)[2]`)
 """
```

Comment:
slight nitpick - I think we should also mention explicitly here that `X` is the `encoded_features`.

Reply:
good call, will do